src/cli/backup/sharepoint_test.go

```diff
@@ -391,51 +387,3 @@ func (suite *SharePointUnitSuite) TestSharePointBackupCreateSelectors() {
 		})
 	}
 }
-
-func (suite *SharePointUnitSuite) TestSharePointBackupDetailsSelectors() {
```

Comment:
Do we have coverage for sharepoint details now? The change in `backup_test` only includes exchange. Can we leave this test in here?

Reply:
This wasn't exercising integration behavior in details, it was just a roundabout way of testing the selectors behavior via mocks, all of which is already covered in other sections. The `backup_test` reference to exchange doesn't exercise exchange in any particular way either.  We primarily need a valid service to stand in place there.

As separate toil, I'd be happy to go through and re-evaluate test coverage after these cases for security, however.  They seem to have grow in a couple different directions across time and could use some re-alignment.